airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py

```diff
@@ -173,7 +173,7 @@
 
 
 class Salesforce:
-    logger = AirbyteLogger()
+    logger = logging.getLogger("airbyte")
```

Comment:
This one seems a bit invasive to pass the logger in dynamically by stream, so this is okay as is I think.